pkgs/development/python-modules/latex2mathml/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, poetry-core
+, setuptools
+, pytestCheckHook
+, pytest-cov
+, multidict
+, xmljson
+}:
+
+buildPythonPackage rec {
+  pname = "latex2mathml";
+  version = "3.75.2";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "roniemartinez";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-i/F1B/Rndg66tiKok1PDMK/rT5c2e8upnQrMSCTUzpU=";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    setuptools  # needs pkg_resources at runtime
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov
```

Comment:
```suggestion
```

Reply:
This is a bit of a problem: pyproject.toml explicitly sets coverage related flags and the build fails if I don't provide pytest-cov. How do you suggest I patch that out? (Or do we just keep pytest-cov after all?)

Reply:
> How do you suggest I patch that out?

Yes